registration-processor/stage-groups/README.md

```diff
@@ -0,0 +1,9 @@
+# Sample Command to build:
+````
+sudo docker build -t  my-stage-group:1.2.0-SNAPSHOT .

```

Comment:
change this example and below to full name format of the docker as specificed in the github workflow files

Reply:
Done.